src/sentry/static/sentry/app/components/gridEditable/gridHeadCell.tsx

```diff
@@ -118,27 +122,54 @@ class GridHeadCell<Column extends GridColumnHeader> extends React.Component<
     );
   }
 
+  renderInBetween = () => {
+    const {
+      isEditing,
+      isLast,
+      openModalAddColumnAt,
+      indexColumnOrder,
+      isColumnDragging,
+    } = this.props;
+
+    if (isLast || !isEditing || isColumnDragging) {
+      return null;
+    }
+
+    return (
+      <React.Fragment>
+        {FLAG_GRID_RESIZABLE && <GridHeadCellResizer isEditing={isEditing} />}
```

Comment:
Will these two components end up in the same small space?

Reply:
I separated them.